solr_plugins/MultiLanguageQueryParser/Makefile

```diff
@@ -0,0 +1,10 @@
+# Compiler
```

Comment:
No idea what this comment is suppoesd to tell us.  Please either remove it or elaborate.